.github/workflows/sync.yml

```
@@ -0,0 +1,101 @@
+name: Sync PR
+
+on:
+  issue_comment:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/sync') }}
+    permissions:
+      pull-requests: write
+      issues: write
+    outputs:
+      should_sync: ${{ fromJSON(steps.validate.outputs.result).shouldSync }}
+      head_sha: ${{ fromJSON(steps.validate.outputs.result).head_sha }}
+      pull_number: ${{ fromJSON(steps.validate.outputs.result).pull_number }}
+      trigger_comment_id: ${{ fromJSON(steps.validate.outputs.result).triggerCommentId }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          sparse-checkout: |
+            .github
+
+      - name: Initial validation and reaction
+        id: validate
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            core.debug(JSON.stringify(context, null, 2));
+            const sync = require('./.github/workflows/sync.js');
+
+            // Validate comment is exactly "/sync"
+            const { comment } = context.payload;
+            if (!sync.shouldSync(comment)) {
+              await sync.updateTriggerComment(context, github, comment.id, '❌ Invalid sync command. Use exactly "/sync"');
+              core.setFailed('Invalid sync command. Use exactly "/sync"');
+              return { shouldSync: false };
+            }
+
+            // Add rocket reaction
+            await github.rest.reactions.createForIssueComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              comment_id: comment.id,
+              content: "rocket",
+            });
+
+            // Update trigger comment with workflow link
+            const workflowRunUrl = `https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}`;
+            await sync.updateTriggerComment(context, github, comment.id, `⏳ [Sync workflow started](${workflowRunUrl})`);
```

```suggestion
            await sync.updateTriggerComment(context, github, comment.id, `⏳ Sync workflow started`);
```

We can remove this since `updateTriggerComment` inserts the link.